addons/huntir/functions/fnc_handleFired.sqf

```diff
@@ -33,15 +33,20 @@ if (_ammo != "F_HuntIR") exitWith {};
         _huntir setPosATL _position;
         [{
             EXPLODE_1_PVT(_this select 0,_huntir);
-            private ["_deltaT"];
             if (isNull _huntir) exitWith {
                 [_this select 1] call CBA_fnc_removePerFrameHandler;
             };
+            private ["_deltaT", "_velocity"];
+            _velocity = velocity _huntir;
             if (damage _huntir > 0) then {
                 _deltaT = ACE_time - (_huntir getVariable [QGVAR(lastTime), ACE_time]);
-                _huntir setVelocity (velocity _huntir vectorAdd [0, 0, -9.8066 * (damage _huntir) * _deltaT]);
+                _velocity = _velocity vectorAdd [0, 0, -9.8066 * (damage _huntir) * _deltaT];
```

Comment:
Fuck, now I finally understand it: you want to slow down the 7.7 m/s fall velocity, which it's clearly too high according to the specs. Yeah, obviously it would be nice to be able to do that through config, but I don't know how either. Maybe somethingrelated to setMass?